ci-operator/config/stolostron/search-indexer/stolostron-search-indexer-main.yaml

```diff
@@ -0,0 +1,79 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.17-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.17-linux
```

Comment:
And is this tag `stolostron_builder_go1.17-linux` or `go1.17-linux`? I'm assuming the latter

Reply:
Correct, the tag `go1.17-linux` stays the same